release/gcb/local.sh

```diff
@@ -1,16 +1,22 @@
 #!/bin/bash
 set -x
+set -o errexit
 
 # shellcheck disable=SC1091
 source gcb_lib.sh
 ROOT=$(cd "$(git rev-parse --show-cdup)" && pwd || return)
 artifacts="$HOME/output/local"
 export NEW_VERSION="fake"
-export DOCKER_HUB='fake/fake'
+export DOCKER_HUB='docker.io/fake'
```

Comment:
Why is this fake? The readme says I need to provide my hub

Reply:
I am constantly building/running this script and don't want to be pushing it to my docker.io hub yet. 
It will be next PR to provide the real hub, etc